build.sbt

```diff
@@ -167,6 +167,22 @@ val mimaFilterSettings = Seq {
     ProblemFilters.exclude[DirectMissingMethodProblem]("scala.reflect.runtime.SynchronizedTypes.defineNormalized"),
     ProblemFilters.exclude[DirectMissingMethodProblem]("scala.reflect.runtime.JavaUniverse.defineNormalized"),
     ProblemFilters.exclude[DirectMissingMethodProblem]("scala.reflect.io.ZipArchive.RootEntry"),
+    ProblemFilters.exclude[ReversedMissingMethodProblem]("scala.runtime.ScalaNumberProxy.clamp"),
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.math.package.clamp"),
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.runtime.RichByte.clamp"),
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.runtime.RichByte$extension.clamp"),
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.runtime.RichChar.clamp"),
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.runtime.RichChar$extension.clamp"),
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.runtime.RichDouble.clamp"),
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.runtime.RichDouble$extension.clamp"),
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.runtime.RichFloat.clamp"),
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.runtime.RichFloat$extension.clamp"),
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.runtime.RichInt.clamp"),
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.runtime.RichInt$extension.clamp"),
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.runtime.RichLong.clamp"),
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.runtime.RichLong$extension.clamp"),
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.runtime.RichShort.clamp"),
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.runtime.RichShort$extension.clamp"),
```

Comment:
I don't think we can do this. I think we need to wait until a 2.14 branch when MiMa is turned off